packages/@ember/-internals/extension-support/lib/container_debug_adapter.ts

```diff
@@ -40,23 +47,22 @@ import { getOwner } from '@ember/-internals/owner';
   @since 1.5.0
   @public
 */
-export default EmberObject.extend({
-  init() {
-    this._super(...arguments);
+export default class ContainerDebugAdapter extends EmberObject {
+  constructor(owner: Owner) {
+    super(owner);
 
-    this.resolver = getOwner(this).lookup('resolver-for-debugging:main');
-  },
+    this.resolver = getOwner(this)!.lookup('resolver-for-debugging:main') as Resolver;
+  }
 
   /**
     The resolver instance of the application
     being debugged. This property will be injected
     on creation.
 
     @property resolver
-    @default null
     @public
   */
-  resolver: null,
+  declare resolver: Resolver;
```

Comment:
Given it's set in the constructor, shouldn't need the `declare` here.

```suggestion
  resolver: Resolver;
```